apps/src/templates/lessonOverview/LessonStandards.jsx

```diff
@@ -222,12 +221,10 @@ class Standard extends PureComponent {
   render() {
     const {standard} = this.props;
     return (
-      <li key={standard.shortcode}>
-        <summary>
-          <span style={styles.standardShortcode}>{standard.shortcode}</span>
-          {' - '}
-          {standard.description}
-        </summary>
+      <li key={standard.shortcode} style={styles.standard}>
+        <span style={styles.standardShortcode}>{standard.shortcode}</span>
+        {' - '}
+        {standard.description}
```

Comment:
I removed the summary wrapper here, which probably shouldn't have been here. this made the gray highlight on hover go away. I added some padding to these elements too.

|  before | after  |
|---|---|
| ![Screen Shot 2021-05-13 at 2 54 34 PM](https://user-images.githubusercontent.com/8001765/118193624-de70d880-b3fc-11eb-9aea-ae419ff35290.png) | ![Screen Shot 2021-05-13 at 3 06 44 PM](https://user-images.githubusercontent.com/8001765/118193641-e466b980-b3fc-11eb-86d6-ef6a6444a35b.png) |